apps/slackbot/utilities/builders.py

```
@@ -18,7 +18,75 @@
 from utilities.slack import actions, forms
 
 # from pymysql.err import ProgrammingError
+SUBMISSION_EXTERNAL_ID = "submission-wait-view"
+SUBMISSION_WAIT_VIEW = View(
+    type="modal",
+    title="Submitting...",
+    external_id=SUBMISSION_EXTERNAL_ID,
+    blocks=[
+		{
```

Ruff (0.15.21)

[warning] Indentation contains tabs

(W191)